src/main/java/com/google/devtools/build/lib/rules/proto/ProtoCompileActionBuilder.java

```diff
@@ -272,12 +271,9 @@ private static String getOutputDirectory(RuleContext ruleContext) {
 
     // Add include maps
     addIncludeMapArguments(
-        getOutputDirectory(ruleContext),
         result,
-        areDepsStrict ? protoInfo.getStrictImportableProtoSourcesImportPaths() : null,
-        protoInfo.getStrictImportableProtoSourceRoots(),
-        protoInfo.getTransitiveProtoSources(),
-        siblingRepositoryLayout);
+        areDepsStrict ? protoInfo.getImportableProtos() : null,
```

Comment:
Mind removing the `siblingRepositoryLayout` variable from line 266? (it's unused now as far as I can tell)